ros2netperf/ros2netperf/verb/server.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import netperf_tool
+
+from ros2netperf.api import add_qos_arguments_to_parser
+from ros2netperf.api import get_qos_profile_from_args
+from ros2netperf.api import print_results
+from ros2netperf.api import print_stats_header
+from ros2netperf.verb import VerbExtension
+
+
+class ServerVerb(VerbExtension):
+    """Run server side of ros2 network performance tester."""
+
+    def __init__(self):
+        self.next_client_id_ = 0
+        self.gid_id_map_ = {}
+
+    def add_arguments(self, parser, cli_name):
+        add_qos_arguments_to_parser(parser)
+
+
+    def main(self, *, args):
+        qos_profile = get_qos_profile_from_args(args)
+
+        runner = netperf_tool.ServerRunner(qos=qos_profile)
+        try:
+            with runner as node:
+                print('---------------------------------------------------------')
+                print('Server running')
+                print(f'\ttopic: {node.get_topic_name()}')
+                print(f'\tqos: {qos_profile}')
```

Comment:
```suggestion
                print(f'    topic: {node.get_topic_name()}')
                print(f'    qos: {qos_profile}')
```

Reply:
Done in https://github.com/ros2/ros2cli/pull/720/commits/0d98122212fed5ca199b81f1d472888873f9ada3